tank_lab_to_nwb/__init__.py

```diff
@@ -1 +1,2 @@
 from .convert_towers_task.towersnwbconverter import TowersNWBConverter
+from .convert_towers_task.tanknwbconverter import TankNWBConverter
```

Comment:
At some point is this `tanknwbconverter` going to replace `towersnwbconverter`?

Reply:
Good point, @weiglszonja. The Tank Lab does more than this experiment, and we should make this robust to potentially having more converters for them in the future, so I'd rather have a single converter and call is something like `TowersNeuropixelNWBConverter`